test/jdk/tools/jpackage/helpers/jdk/jpackage/test/JPackageCommand.java

```diff
@@ -643,6 +643,7 @@ private Executor createExecutor() {
             exec.setToolProvider(JavaTool.JPACKAGE);
         } else {
             exec.setExecutable(JavaTool.JPACKAGE);
+            exec.setTmpDir(System.getProperty("java.io.tmpdir"));
```

Comment:
This would work only on Windows. I'd put corresponding `if` around this statement to avoid future confusion.

Reply:
That makes sense, should this be here in JPackageCommand.CreateExecutor() or in Executor.setTempDir() (maybe renamed setWinTempDir ?)